custom_components/config_check/__init__.py

```diff
@@ -119,6 +124,13 @@ def clear_result(string):
     for clear in clear_out:
         string = string.replace(clear, "")
 
-    string = string.replace("Failed config", "**Failed config**")
+    """Remove HA colorlog install attempt, regardless of version """

```

Comment:
```suggestion
    # Remove HA colorlog install attempt, regardless of version

```

`"""` are used for docstring/multiline comments, `# ` are used for single line